File: js/src/0.0-Basics/0.3-build-up-logical-thinking/0.3.26-pattern.js
function log(val) {
    process.stdout.write(String(val));
}

function pattern_26() {
    let row = 4;
    let column = 4;

    for (let y = 0; y < row; y++) {
        for (let x = 0; x < column; x++) {
            if (y == 0 || y == row - 1 || x == 0 || x == column - 1) {
                log('*');
            } else {
                log(' ');
            }
        }
        console.log();
    }
}

console.clear();
pattern_26();

File: python/00-Basics/00.03-build-up-logical-thinking/0.3.26-pattern.py
from utils.clear import clear

def pattern_26():
    row = 10
    column = 10

    for i in range(0, row):
        for j in range(0, column):
            if i == 0 or i == row - 1 or j == 0 or j == column - 1:
                print("*", end="")
            else:
                print(" ",end="")
        print()


clear()
pattern_26()
